docs/addons/index.md

```diff
@@ -8,26 +8,27 @@ Within the context of the `ssp-amazon-eks` framework, an add-on is abstracted as
-| [`VeleroAddOn`](./velero.md) | Adds [Velero](https://velero.io/) to the EKS Cluster |
+| Add-on                                                                           | Description                                                                                                                           |
+|----------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------|
+| [`AppMeshAddOn`](./app-mesh)                                                     | Adds an AppMesh controller and CRDs (pending validation on the latest version of CDK)                                                 |
+| [`ArgoCDAddOn`](./argo-cd)                                                       | Provisions Argo CD into your cluster.                                                                                                 |
+| [`AWS Load Balancer Controller`](./aws-load-balancer-controller)                 | Provisions the AWS Load Balancer Controller into your cluster.                                                                        |
+| [`CalicoAddOn`](./calico)                                                        | Adds the Calico 1.7.1 CNI/Network policy engine                                                                                       |
+| [`ClusterAutoscalerAddOn`](./cluster-autoscaler)                                 | Adds the standard cluster autoscaler ([Karpenter](https://github.com/awslabs/karpenter) is coming)                                    |
+| [`ContainerInsightsAddOn`](./container-insights)                                 | Adds Container Insights support integrating monitoring with CloudWatch                                                                |
+| [`CoreDnsAddOn`](./coredns.md)                                                   | Adds CoreDNS Amazon EKS add-on. CoreDNS is a flexible, extensible DNS server that can serve as the Kubernetes cluster DNS             |
+| [`KubeProxyAddOn`](./kube-proxy.md)                                              | Adds kube-proxy Amazon EKS add-on. Kube-proxy maintains network rules on each Amazon EC2 node                                         |
+| [`MetricsServerAddOn`](./metrics-server.md)                                      | Adds metrics server (pre-req for HPA and other monitoring tools)                                                                      |
+| [`ExternalDnsAddOn`](./external-dns)                                             | Adds [External DNS](https://github.com/kubernetes-sigs/external-dns) support for AWS to the cluster, integrating with Amazon Route 53 |
+| [`Keptn`](https://github.com/keptn-sandbox/keptn-ssp-addons/blob/main/README.md) | [Keptn](https://www.keptn.sh) AddOn                                                                                                   |
```

Comment:
Formatting changed for the whole block. It is a bit hard to see what changed specifically. Would you mind changing just the Keptn portion?

Reply:
Fixed this ...